suffuse/process.go

```diff
@@ -16,12 +17,13 @@ type ExecResult struct {
   Stdout []byte
 }
 
-func cwd() Path        { return MaybePath(os.Getwd()) }
+func cwd() Path           { return MaybePath(os.Getwd()) }
+func OsStderr() io.Writer { return os.Stderr             }
 
-func (x ExecResult) OneLine() string { return x.Lines().JoinWords()  }
-func (x ExecResult) Lines() Lines    { return BytesToLines(x.Stdout) }
-func (x ExecResult) Slurp() string   { return string(x.Stdout)       }
-func (x ExecResult) Success() bool   { return x.Err == nil           }
+func (x ExecResult) OneLine() string  { return x.Strings().JoinWords()  }
+func (x ExecResult) Strings() Strings { return BytesToStrings(x.Stdout) }
```

Comment:
These weren't missed renames. In fact I had to change them back to Lines, since I renamed all at once. The method is supposed to be called "Lines" because it's the lines of the process output. The data structure it returns is Strings. The way I had it somewhere (can't keep the PRs and branches straight) is `func (x ExecResult) Lines() Strings` which is as intended.